README.md

```diff
@@ -1,6 +1,261 @@
+| tcdm_rsp | data | io_data_tcdm_rsp_0_bits_data | 64| In| The response data from the read request. |
+| tcdm_rsp | valid | io_data_tcdm_rsp_0_valid | 1| In| The signal indicates if this response is valid. |
+| | . | . | . | . | The tcdm_rsp ports number is the same as tcdm_req. |
+| streamer2accelerator | data | io_data_streamer2accelerator_data_0_bits | First read FIFO width| Out | The data for the acceleratorX input. |
+| streamer2accelerator | valid | io_data_streamer2accelerator_data_0_valid | 1| Out| The signal indicates if this data is valid. |
+| streamer2accelerator | ready | io_data_streamer2accelerator_data_0_ready | 1| In| The signal indicates if the acceleratorX is ready for this data (has used this data already). |
+| | . | . | . | . | The streamer2accelerator ports number is the same as data reader number. The index 0 corresponds to the first input data and 2 for second input data and so on. |
+| accelerator2streamer | data | io_data_accelerator2streamer_data_0_bits | First write FIFO width| In| The data from the acceleratorX output. |
+| accelerator2streamer | valid | io_data_accelerator2streamer_data_0_valid | 1| In| The signal indicates if this data is valid. |
+| accelerator2streamer | ready | io_data_accelerator2streamer_data_0_ready | 1| Out| The signal indicates if the data writer is ready for taking in this data (not full). |
+| | . | . | . | . | The accelerator2streamer ports number is the same as data writer number. The index 0 corresponds to the first output data and 2 for second output data (if any) and so on.|
+| | | | | | | |
+
+### CSR definition
+| Address | CSR name             | Notes                               |
+|---------|--------------------------|-------------------------------------|
+| offset* + [0..temporalDim - 1]   |    temporalLoopBoundCSRs     | temporal loop bound for each temporal dimension.           |
+| offset + temporalDim + [0..temdataMoverNum * temporalDimporalDim - 1]   |    temporalLoopSrtidesCSRs     | temporal loop strides for each temporal dimension and for each data mover.          |
+| offset + temporalDim +  dataMoverNum * temporalDim + [0..spatialDim.sum - 1]   |    spatialLoopSrtidesCSRs     | spatial loop strides for each data mover and for corresponding spatial dimension. The spatial dimension for each data mover can be different. It depends on the accelerator.        |
+| offset + temporalDim +  dataMoverNum * temporalDim + spatialDim.sum + [0..dataMoverNum - 1]   |    basePtrCSRs     |  base pointers for each data mover.          |
+
+*offset is defined by the SNAX core. A more detailed explanation of what are these configurations can be found at `StreamerTop.scala`.
```

Comment:
maybe don't use the * here to reference the offset, this is a bit confusing, since it might be interpreted as a pointer dereference operator and it is used as  a multiplication sign further down the line

Reply:
Maybe just put the explanation of the offset before the table

Reply:
Ok!